operator/src/main/java/org/keycloak/operator/controllers/KeycloakDeploymentDependentResource.java

```diff
@@ -183,7 +188,7 @@ public StatefulSet desired(Keycloak primary, Context<Keycloak> context) {
         };
     }
 
-    private void configureCache(StatefulSet deployment, Container kcContainer, CacheSpec spec, KubernetesClient client, WatchedResources watchedResources) {
+    private void configureCache(StatefulSet deployment, Container kcContainer, CacheSpec spec, KubernetesClient client, WatchedResources watchedResources, TreeSet<String> allConfigMaps) {
```

Comment:
This does not need `watchedResources`, no?

Reply:
It should be there - we're missing a test for that. Corrected and expanded the existing test case.

Reply:
Thanks for the update.

It would seem that actually neither `client` nor `watchedResources` is used in `configureCache`.

Reply:
Sorry, I should've been more clear.